articles/configuration/msbuild.md

```diff
@@ -1,3 +1,41 @@
-# MSBuild configuration
+---
+title: MSBuild Configuration
+author: uniqueiniquity
+ms.author: belichtm
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net
+uid: msbuild
+---
 
-TODO
+# MSBuild Configuration
+
+## Props and Targets
+
+There are two different sources of information that tell MSBuild how to handle TypeScript. The first, `Microsoft.TypeScript.Default.props`, establishes some default TypeScript compilation settings. The second, `Microsoft.TypeScript.targets`, gives instructions on how to handle TypeScript files as part of the building and cleaning processes. To include these in your project, you will need to (in most cases) import `Microsoft.TypeScript.Default.props` at the beginning of your project file and `Microsoft.TypeScript.targets` at the end, as shown [in the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/integrating-with-build-tools.html#msbuild).
+
+> [!NOTE]
+> ASP.NET Core projects implicitly include these files, so an ASP.NET Core project file will not need to import them.
+
+## TypeScript Tools Version
+Since a particular machine may have multiple versions of TypeScript installed, MSBuild allows you to set the TypeScriptToolsVersion property in your project file to specify which version to use.
```

Comment:
Could you give a code example or link to an existing TypeScript doc page with the code example?

Reply:
Added.